packages/cashier/src/Sass/cashier.scss

```diff
@@ -1122,6 +1122,79 @@
     }
 }
 
+.cashier-recent-transaction-wrapper {
+    max-width: 256px;
+    @include mobile {
+        max-width: 328px;
+        margin: 0 auto;
+    }
+
+    .cashier-recent-transaction {
+        overflow: auto;
+        margin-bottom: 1rem;
+        min-width: 256px;
+        padding: 1.6rem 2.4rem;
+        border-radius: 8px;
+        background-color: var(--general-section-1);
+        color: var(--text-general);
+        font-size: var(--text-size-xxs);
```

Comment:
Why not use `Text` component?

Reply:
I should have deleted these props at all, thanks!